src/frontend/js/order/sagas.js

```diff
@@ -13,26 +14,23 @@ import {
     fetchNewOrder as fetchNewOrderApi,
 } from './api';
 
-export const newOrder = (fetchNewOrder) => function* newOrderSaga({ payload }) {
-    const jwt = yield select(jwtSelector);
-    const {
-        error,
-        order,
-    } = yield call(fetchNewOrder, payload, jwt);
+export const newOrderSaga = fetchNewOrderSaga => function* ({ payload }) {
```

Comment:
What about `orderActionsf.failure` action ? It's no longer dispatched with this change